bluetooth-media: fix problem that a2dp can not be stopped

gzh-terry opened the pull request:
Rootcause: After media sending A2DP_CTRL_CMD_STOP, still remain some bytes in circlebuf and too short to send, result in a2dp can not go to stop.

chengkai15 commented on service/profiles/a2dp/source/a2dp_source_audio.c:
```diff
@@ -249,7 +249,7 @@ static void a2dp_source_audio_handle_timer(service_timer_t* timer, void* arg)
 
     /* Handle stream underflow */
-    if (circbuf_used(&stream->stream_pool) == 0) {
+    if (circbuf_used(&stream->stream_pool) < stream->stream_interface->get_min_frame_size()) {
```
if circbuf_used 0, it would nerver do ticks++

gzh-terry replied:
It will, cuz `0 < get_min_frame_size()`

gzh-terry replied:
Change AAC min frame size to 1.

huangyulong3 commented on service/profiles/a2dp/source/a2dp_source_audio.c:
```diff
@@ -249,7 +249,7 @@ static void a2dp_source_audio_handle_timer(service_timer_t* timer, void* arg)
 
     /* Handle stream underflow */
-    if (circbuf_used(&stream->stream_pool) == 0) {
+    if (circbuf_used(&stream->stream_pool) < stream->stream_interface->get_min_frame_size()) {
```
When the buffer size is smaller than a frame size, is it not necessary to clean up the `stream_pool's circbuf`?

gzh-terry replied:
We need to cleanup circbuf since all data left is outdated in the next stream.

gzh-terry committed:
bluetooth-media: fix problem that a2dp can not be stopped when media send A2DP_CTRL_CMD_STOP

bug: v/65796

Rootcause: After media sending A2DP_CTRL_CMD_STOP, still remain some bytes in circlebuf and too short to send, result in a2dp can not go to stop.

Signed-off-by: Zihao Gao <gaozihao@xiaomi.com>
Signed-off-by: zhangyuan20 <zhangyuan20@xiaomi.com>
